### PR TITLE
Created module to publish HTML to DOM

### PR DESCRIPTION
**Pull request by croninsw**
Removed comma from database.json file, left secret message on main.js, removed Herobrine

### CODE REVIEW

File: src/scripts/printToDOM.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const printToDOM = (object) => {
+    document.querySelector("#this_can_be_called_whatever_for_now").innerhtml += object
```
Comment (naienko) [inline src/scripts/printToDOM.js]: shouldn't this function also take the location inside queryselector as an argument?

Comment (croninsw) [reply src/scripts/printToDOM.js]: w/e panyyaaaa

**Review by MoSilvera — APPROVED** [review]
👍 Much skill, such success

**Comment by naienko** [comment]
:+1:

**Review by naienko — APPROVED** [review]
:+1:

**Pull request merged**